hack/test/day-two/config.yaml

```diff
@@ -0,0 +1,31 @@
+charts:
+  loki:
+    namespace: loki
+    repo: https://grafana.github.io/helm-charts
+    chart: loki-stack
+    valuesPath: hack/test/day-two/loki-values.yaml      
+
+  metallb:
+    namespace: metallb
+    repo: https://metallb.github.io/metallb
+    chart: metallb
+    valuesPath: hack/test/day-two/metallb-values.yaml      
+
+  ingress-nginx:
+    namespace: ingress
+    repo: https://kubernetes.github.io/ingress-nginx
+    chart: ingress-nginx
+    depends:
+      - metallb
+
+  rook:
+    namespace: rook-ceph
+    repo: https://charts.rook.io/release
+    chart: rook-ceph
+
+  ceph:
```

Comment:
it makes now sense to update the `e2e-qemi-csi` tests and use the charts for install, and move the tests somewhere with day2 tests

Reply:
https://github.com/talos-systems/talos/blob/master/hack/test/e2e.sh#L228

Reply:
Once I've confirmed the day-two tests work and cover the same case, I will remove the `e2e-qemu-csi` tests. I just didn't want to break anyone using them in the mean time :)